File: examples/discord/jda/slash/src/main/java/dev/triumphteam/slash/example/ExampleCommandGroup.java
package dev.triumphteam.slash.example;

import dev.triumphteam.cmd.core.annotations.Command;
import dev.triumphteam.cmd.slash.sender.SlashSender;
import net.dv8tion.jda.api.entities.User;

@Command("group")
public class ExampleCommandGroup {

    @Command("test")
    public class Group {

        @Command("first")
        public void first(final SlashSender sender) {

        }

        @Command("second")
        public void second(final SlashSender sender, final User user) {

        }
    }
}

File: examples/discord/kord/slash/src/main/kotlin/dev/triumphteam/kord/example/commands/ExampleCommandGroup.kt
package dev.triumphteam.kord.example.commands

import dev.kord.core.entity.User
import dev.triumphteam.cmd.core.annotations.Command
import dev.triumphteam.cmds.kord.sender.SlashSender

@Command("group")
public class ExampleCommandGroup {
    @Command("test")
    public inner class Group {
        @Command("first")
        public suspend fun first(sender: SlashSender) {
            sender.reply("Command sent was /group test first")
        }

        @Command("second")
        public suspend fun second(sender: SlashSender, user: User) {
            sender.reply("Command sent was /group test second <$user>")
        }
    }
}
